en/flight_controller/pixracer.md

```diff
@@ -52,13 +52,24 @@ The Pixracer is designed to use a separate avionics power supply. This is necess
 One of the main features of the board is its ability to use Wifi for flashing new firmware, system setup and in-flight telemetry. 
 This frees it of the need of any desktop system.
 
-::note ToDo
-Setup and telemetry are already available, firmware upgrade is already supported by the default bootloader but not yet enabled
-:::
-
 * [ESP8266 Wifi](../telemetry/esp8266_wifi_module.md)
 * [Custom ESP8266 MAVLink firmware](https://github.com/dogmaphobic/mavesp8266)
 
+:::note
+Firmware upgrade is not yet enabled over WiFi (it is supported by the default bootloader but not yet enabled).
+Setup and telemetry are supported.
+:::
+
+
+## External Telemetry
+
+The flight controller also supports telemetry via external Wi-Fi or radio telemetry modules connected to the `TELEM1` or `TELEM2` ports.
```

Comment:
@potaito Thanks very much. I'm going to merge this so I can add an image below here.

Note, the change from a WiFi and Telem2 focus to an External telemetry focus. Looking at the diagram this refers to how you can connect either radios and Wifi to Telem1 OR Telem2. 

The only error in the diagram is that it says everything will be plug and play, while for TELEM2  you discovered it isn't - so that is the bit we need to surface. But I don't want to just focus on that issue.

I have not confirmed that TELEM1 is plug and play :-)

Reply:
Thanks for the clarification. You're right, it's odd to focus on this one example. Thanks for reformatting the text.